tests/rbac/common/user/reject_manager_good_test.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2018 Contributors to Hyperledger Sawtooth
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+import pytest
+import logging
+
+from rbac.common import protobuf
+from rbac.common.protobuf.rbac_payload_pb2 import RBACPayload
+from rbac.common.user.user_manager import UserManager
+from rbac.common.user.manager import Manager
+from rbac.common.user.reject_manager import RejectUpdateUserManager
+from tests.rbac.common.manager.test_base import TestBase
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.mark.user
```

Comment:
Are we going to ever need this level of granularity? I am confused about marking tests by domain entity as well as interleaving unit and integration tests in the same file.

Generally, I prefer unit tests in one location, integration tests in another, e2es..., smoke..., perf.. Doing so consolidates the setup and teardown fixturing that increases in complexity alongside the type of test.

Regarding test markers: I worry that too many markers/annotations adds confusion rather than keeps things simple. The pathing of the directories will reveal domain entity coverage in coverage reports, and I wonder how one would mark a teast touching users, tasks, and roles.

Reply:
Very well. We can revisit tests later if/when need to reorg comes up. Most important is that they are being written in the first place.